server/lib/payments.js

```diff
@@ -494,7 +494,7 @@ export const sendOrderProcessingEmail = async order => {
       OrderId: order.id,
     };
     data.instructions = stripHTML(instructions).replace(/{([\s\S]+?)}/g, (match, key) => {
-      if (key && formatValues[key]) {
+      if (key && !isNil(formatValues[key])) {
```

Comment:
This is to prevent displaying variable names when the value is missing (eg. `make a transfer to: {account}`)

I've made a similar fix in https://github.com/opencollective/opencollective-frontend/pull/6309.